diff --git a/Makefile b/Makefile
index e692e9d..15e051f 100644
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ BIN=bin
 
 LIB=$(shell pwd)/urb
 
-RM=rm -f
+RM=rm -rf
 ifneq ($(UNAME),FreeBSD)
 CC=gcc
 CXX=g++
@@ -459,6 +459,7 @@ debinstall:
 
 clean:
 	$(RM) $(VERE_OFILES) $(BIN)/urbit urbit.pkg $(VERE_DFILES)
+	$(RM) debian/files debian/urbit*
 
 # 'make distclean all -jn' ∀ n>1 still does not work because it is possible
 # Make will attempt to build urbit while it is also cleaning urbit..
@@ -468,5 +469,6 @@ distclean: clean $(LIBUV_MAKEFILE)
 	$(MAKE) -C outside/ed25519 clean
 	$(MAKE) -C outside/anachronism clean
 	$(MAKE) -C outside/scrypt clean
+	$(RM) debian/files debian/urbit*
 
 .PHONY: clean debbuild debinstalldistclean etags osxpackage tags
diff --git a/include/noun/allocate.h b/include/noun/allocate.h
index 26423f9..d6e1aff 100644
--- a/include/noun/allocate.h
+++ b/include/noun/allocate.h
@@ -19,7 +19,7 @@
   **/
     /* u3a_bits: number of bits in word-addressed pointer.  29 == 2GB.
     */
-#     define u3a_bits  29
+#     define u3a_bits  28
 
     /* u3a_page: number of bits in word-addressed page.  12 == 16Kbyte page.
     */
