diff --git a/Makefile b/Makefile
index 8be9e41..94861d6 100644
--- a/Makefile
+++ b/Makefile
@@ -85,7 +85,6 @@ CWFLAGS=-Wall \
         -Wextra \
         -Wno-sign-compare \
         -Wno-unused-parameter \
-        -Werror \
         -Wno-error=missing-field-initializers
 
 ifdef NO_SILENT_RULES
diff --git a/i/n/a.h b/i/n/a.h
index 9a42bdc..daf7107 100644
--- a/i/n/a.h
+++ b/i/n/a.h
@@ -13,7 +13,7 @@
   **/
     /* u3a_bits: number of bits in word-addressed pointer.  29 == 2GB.
     */
-#     define u3a_bits  29
+#     define u3a_bits  28
 
     /* u3a_page: number of bits in word-addressed page.  12 == 16Kbyte page.
     */
